src/test/resources/config-example.yaml

```diff
@@ -0,0 +1,18 @@
+input:
+  shakespeare:
+    format: bigquery
+    options:
+      table: "bigquery-public-data.samples.shakespeare"
+query:
+  wordspercorpus:
+    sql: >
+      SELECT
+      corpus, count(*) distictwords, sum(word_count) numwords
+      FROM shakespeare
+      GROUP by corpus
+      ORDER BY numwords DESC
+output:
+  wordspercorpus:
+    format: avro
+    path: gs://yadavaja-sandbox-tsell-us/output/wordspercorpus/
+    mode: Overwrite
```

Comment:
This example isn't required by any of the unit tests, I include it to make trying it easier without having to craft one version. We can remove this later.

Reply:
agree, should remove before merging